packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -1988,6 +1979,7 @@ function ScatterplotViz(props: VisualizationProps<Options>) {
         ]}
       />
 
+      <h1>foo</h1>
```

Comment:
Don't suspect we want this in our legends 😄

Reply:
Removed!